base-defconfig

```diff
@@ -237,6 +237,7 @@ CONFIG_SND_SOC_ACPI_INTEL_MATCH=m
 # DRM
 CONFIG_DRM=m
 CONFIG_DRM_I915_ALPHA_SUPPORT=y
+CONFIG_DRM_XE=m
```

Comment:
This should be good. We also need "CONFIG_DRM_XE_DISPLAY=y" but that's anyways the default.

Reply:
confirmed, ONFIG_DRM_XE_DISPLAY=y is in generated .config file

Reply:
Looks CONFIG_DRM_I915_ALPHA_SUPPORT is removed from i915 driver.